data.js

```diff
@@ -3718,6 +3718,19 @@ module.exports = [
     url: "https://github.com/avoidwork/csv.js",
     source: "https://raw.githubusercontent.com/avoidwork/csv.js/master/lib/csv.js"
   },
+  {
+    name: "bassoon.js",
+    tags: ["oboe", "clarinet", "sax", "ajax", "xhr", "fetch", "stream", "json", "array", "object", "front-end"],
+    description: "Library for progressive parsing of ajax responses. Provides notification of objects found without waiting for the request to complete.",
+    url: "https://github.com/paulsmithkc/bassoon#readme",
+    github: "paulsmithkc/bassoon",
+    source: [
+     "https://raw.githubusercontent.com/paulsmithkc/bassoon/master/src/bassoon.mjs",
+     "https://raw.githubusercontent.com/paulsmithkc/bassoon/master/src/Emitter.mjs",
+     "https://raw.githubusercontent.com/paulsmithkc/bassoon/master/src/Parser.mjs",
+     "https://raw.githubusercontent.com/paulsmithkc/bassoon/master/src/bassoon-worker.js"
```

Comment:
This fourth link gives me a 404 Not Found. I'm guessing this existed before and broke sometime since the PR was submitted. (Sorry, this repository no longer has any particularly active maintainers.) To have the PR accepted (...when I get around to it...), please update these source links.

Reply:
The file extension changed from .js to .mjs